interpreter_test.py

```diff
@@ -137,7 +137,15 @@ def test_method_returns_string_token_when_passed_a_longer_string(self):
         tokeniser = Tokeniser("<<a lot of text>>")
         self.assertEqual(tokeniser.create_tokens(), [{"STRSTART" : "<<"}, {"STRING_CONTENT" : "a lot of text"}, {"STRSTOP" : ">>"}])
         
-                
+    def test_method_returns_comparison_token_for_owl_operator(self):
+        from forest import Tokeniser
+        tokeniser = Tokeniser("OvO")
+        self.assertEqual(tokeniser.create_tokens(), [{"EQUALS" : "OvO"}])        
 
+    def test_method_returns_comparison_token_and_bool_for_owl_operator_with_bools(self):
+        from forest import Tokeniser
+        tokeniser = Tokeniser("true^OvO^false")
+        self.assertEqual(tokeniser.create_tokens(), [{"BOOLEAN" : "true"}, {"EQUALS" : "OvO"}, {"BOOLEAN" : "false"}])        
```

Comment:
Could you add a test for comparing two strings, which is what we're going to do the grammar rule for?